docs/proposals/formatted-output.md

```diff
@@ -89,7 +89,7 @@ $ oras pull $REGISTRY/$REPO:$TAG --format json
 #### Example: pull multiple files
 
 ```shell
-$ oras pull $REGISTRY/$REPO:$TAG --format json {{ toPrettyJson }}
+$ oras pull $REGISTRY/$REPO:$TAG --format json {{ toPrettyJson }} 
```

Comment:
I'm a bit confused on what this is supposed to mean because it doesn't work as a cut and paste command
```suggestion
$ oras pull $REGISTRY/$REPO:$TAG --format json # toPrettyJson
```

Reply:
Good catch. It should be `--format '{{jsonPretty .}}'`